booleans.js

```diff
@@ -0,0 +1,79 @@
+const negate = (a) => {
+  return !a;
+};
+
+const both = (a, b) => {
+  return a && b;
+};
+
+const either = (a, b) => {
+  return a || b;
+};
+
+const none = (a, b) => {
+  return !a && !b;
+};
+
+const one = (a, b) => {
+  return !a && b || !b && a;
+};
+
+const truthiness = (a) => {
+  return !!a;
+};
+
+const isEqual = (a, b) => {
+  return a === b;
+};
+
+const isGreaterThan = (a, b) => {
+  return a > b;
+};
+
+const isLessThanOrEqualTo = (a, b) => {
+  return a <= b;
+};
+
+const isOdd = (a) => {
+  return a % 2 === 1
+};
+
+const isEven = (a) => {
+  return a % 2 === 0
+};
+
```

Comment:
Good work getting all of these in one line of code! A lot of people would assume you needed to use `if/else if/else` to solve these.